opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -854,7 +853,17 @@ namespace Opm {
         }
     }
 
-    // static member function
+    
+    const MessageContainer& Eclipse3DProperties::getMessageContainer() const {
```

Comment:
>messages added by GridProperties<double> will be lost space.

I agree that's a fault. need more consideration.

> and then pass a reference to that instance to the GridProperty<T>

I'm not quite clear how to do this, should I add one more argument to the methods?
```
 GridProperty<int>&      getIntGridProperty()
 GridProperty<double>&    getDoubleGridProperty()

```